projects/js-packages/components/components/admin-page/index.jsx

```diff
@@ -32,14 +32,20 @@ const AdminPage = props => {
 		showHeader,
 		showFooter,
 		showBackground,
+		header,
 	} = props;
 	const rootClassName = classNames( styles[ 'admin-page' ], {
 		[ styles.background ]: showBackground,
 	} );
 
 	return (
 		<div className={ rootClassName }>
-			{ showHeader && (
+			{ showHeader && !! header && (
+				<Container horizontalSpacing={ 5 }>
+					<Col>{ header }</Col>
+				</Container>
+			) }
```

Comment:
You don't need to create a new set of components. It could be more cleaner:

```es6
{ showHeader && (
  <Container horizontalSpacing={ 5 }>
    <Col>{ Boolean(header) ? header : <JetpackLogo />  }</Col>
  </Container>
) }
```

Reply:
💯

Reply:
`Boolean( header )` seems to be redundant

Reply:
@retrofox yes, in this case, it is, sorry ... normally I use it with `&&` to avoid this case: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx